arch/arm/src/sama5/sam_tsd.c

```diff
@@ -1655,7 +1755,7 @@ int sam_tsd_register(struct sam_adc_s *adc, int minor)
 
   /* Initialize the touchscreen device driver instance */
 
-  priv->adc = adc; /* Save the ADC device handle */
+  priv->adc     = adc;               /* Save the ADC device handle    */
```

Comment:
why this is changed?

Reply:
no reason so keyboard error I guess...but checkpatch.sh didn't find it :(

Have corrected.